src/core/path.c

```diff
@@ -56,7 +56,6 @@ QuicPathRemove(
 
 #if DEBUG
     if (Path->DestCid) {
-        CXPLAT_DBG_ASSERT(Path->DestCid->AssignedPath != NULL);
```

Comment:
Can you explain why this is no longer valid?

Reply:
Because it was possible for the path to not be assigned yet by the time if was getting removed. So this check would fail if that ended up being the case.